src/library.scss

```diff
@@ -36,4 +36,6 @@
 @use "./components/share/modal/share-modal";
 @use "./components/modal";
 @use "./components/badge";
+@use "./components/icon";
 @use "./assets/fonts/Marianne/Marianne-font.css";
+@use "@fontsource/material-icons";
```

Comment:
Our style.css is becoming huge, we check if we find a way to decrease it or if we have no choice